### PR TITLE
Polish docs

### PR DESCRIPTION
**Pull request by felixfbecker**
- Use ES6 in docs, as supported by our minimal version (Node 4). That means arrow functions and `const`.
- Make model names PascalCase. Models are classes in v4 and the name becomes the class name, PascalCase is the convention for that.
- Make indentation consistently 2 spaces.

### CODE REVIEW

**Comment by felixfbecker** [comment]
@sushantdhiman the docs used both PascalCase and camelCase model names, it's not consistent. I agree that attributes should always be camelCase, but _classes_ should always be PascalCase. It would be better than to always specify an alias.

**Comment by sushantdhiman** [comment]
@felixfbecker In that case please update docs with proper foreign key names as well, because even though models are now Pascal cased, auto generated foreign keys aren't.

**Comment by felixfbecker** [comment]
Will do in a second PR, removed the commit for now

File: docs/associations.md [hunk 1]
```diff
@@ -244,10 +244,17 @@ Person.belongsToMany(Person, { as: 'Children', through: 'PersonChildren' })
...
+const User = sequelize.define('user', {})
+const Project = sequelize.define('project', {})
+const UserProjects = sequelize.define('userProjects', {
     status: DataTypes.STRING
+=======
+const User = sequelize.define('User', {})
+const Project = sequelize.define('Project', {})
+const UserProjects = sequelize.define('UserProjects', {
+  status: DataTypes.STRING
+>>>>>>> 1a74b8b3... fixup
```
Comment (sushantdhiman) [inline docs/associations.md]: Opps

Comment (felixfbecker) [inline docs/associations.md]: 👀

File: docs/hooks.md [hunk 2]
```diff
@@ -245,23 +245,25 @@ Model.destroy({ where: {username: 'Tom'}} /*whereClause argument*/)
...
+```js
 // Bulk updating existing users with updatesOnDuplicate option
-Users.bulkCreate([{ id: 1, isMemeber: true},
-                 { id: 2, isMember: false}],
-                 { updatesOnDuplicate: ['isMember']})
+Users.bulkCreate([
+  { id: 1, isMemeber: true},
+  { id: 2, isMember: false},
+  { updatesOnDuplicate: ['isMember']
+});
```
Comment (sushantdhiman) [inline docs/hooks.md]:
```js
Users.bulkCreate([
 { id: 1, isMemeber: true},
 { id: 2, isMember: false}
], {
 updatesOnDuplicate: ['isMember']
});

**Pull request merged**